src/hyperfine/timer/unix_timer.rs

```diff
@@ -55,6 +55,7 @@ fn get_cpu_times() -> CPUTimes {
             + i64::from(result.ru_utime.tv_usec),
         system_usec: i64::from(result.ru_stime.tv_sec) * MICROSEC_PER_SEC
             + i64::from(result.ru_stime.tv_usec),
+        max_rss_byte: i64::from(result.ru_maxrss),
```

Comment:
From `man 2 getrusage`: "in kilo-bytes"
```
       ru_maxrss (since Linux 2.6.32)
              This is the maximum resident  set  size  used  (in  kilo‐
              bytes).   For  RUSAGE_CHILDREN,  this is the resident set
              size of the largest child, not the maximum  resident  set
              size of the process tree.
```

Reply:
Also, I'm not sure what to make of the "For  RUSAGE_CHILDREN,  this is the resident set size of the largest child, not the maximum  resident  set size of the process tree" comment. Does this mean that we would not get the total amount of memory used by the tree
```
+ /bin/sh
+--+ command-to-be-benchmarked
```
but rather the memory usage of EITHER the shell or the command-to-be-benchmarked?!

Reply:
I wonder now if libc::wait4 wouldn't give us better information.